Rubberduck.RibbonDispatcher/Rubberduck.ExcelRibbon2013/RubberduckRibbon.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System;
+using System.IO;
+using System.Reflection;
+using System.Runtime.InteropServices;
+using System.Diagnostics.CodeAnalysis;
+using Microsoft.Office.Core;
+
+using Rubberduck.RibbonDispatcher;
+using Rubberduck.RibbonDispatcher.Abstract;
+using Rubberduck.RibbonDispatcher.Concrete;
+
+namespace Rubberduck.RibbonDispatcher.Excel2013 {
+    [ComVisible(true)][CLSCompliant(true)]
+    public class RubberduckRibbon : AbstractRibbon, IRibbonExtensibility {
+
+        public RubberduckRibbon() : base() {;}
+
+        public string GetCustomUI(string RibbonID) => GetResourceText("Rubberduck.ExcelRibbon2013.RubberduckRibbon.xml");
+
+        [SuppressMessage("Microsoft.Design", "CA1061:DoNotHideBaseClassMethods", 
+            Justification = "This is simply how the Ribbon design works to share the Callback methods between multiple Office products & documents.")]
+        [CLSCompliant(false)]
+        public override void OnRibbonLoad(IRibbonUI ribbonUI) {
+            base.OnRibbonLoad(ribbonUI);
+
+            _standardButtonsViewModel = new StandardButtonsViewModel(RibbonFactory, (s,e) => _customButtonsViewModel.SetVisible(e.IsPressed));
+            _customButtonsViewModel   = new CustomButtonsViewModel(RibbonFactory);
+        }
+
+        StandardButtonsViewModel     _standardButtonsViewModel;
+        CustomButtonsViewModel       _customButtonsViewModel;
+
+        #region Helpers
+
+        private static string GetResourceText(string resourceName) {
```

Comment:
soo ... it actually is relevant for other Ribbons. Why isn't it in an accessible helper? Especially considering it poses as a stateless method?

Reply:
Only 24 hours in a day - It's robo-code, that I have not (yet) touched in any way.